Reworking of includes and layouts episodes as per #208 (#242)
carpentries-incubator/jekyll-pages-novice: _episodes/includes.md

But the principles behind reusing content apply regardless of
the particular framework being used to build the site.

## Reusing Site Navigation 

Let's look at an example of how we can create a block of common content
and reuse it in multiple pages on our site.
Websites typically have some navigation links at the top of each page to help visitors navigate to various portions of the site.
To make these links appear above every page on our site,

> we could add the same code above to each Markdown file in our repository.

Reviewer: ```suggestion
we could add the same code immediately after the YAML header of each Markdown file in our repository.
```

> But if we wanted to adjust the - adjust the navigation link target, add a new navigation link, remove a link, etc. -

Reviewer: ```suggestion
But if we wanted to adjust the menu - adjust the navigation link target, add a new navigation link, remove a link, etc. -
```

> we would need to make the same adjustment on every page.

Reviewer: ```suggestion
we would need to make the same adjustment on every page.
This is both time-consuming and error-prone: 
it would be easy to accidentally mistype a link or forget to update one of the files.
```

Instead, we can go some way to avoid this hassle by using

> some magic that Jekyll provides: `include` tags.

Reviewer: ```suggestion
some magic that Jekyll provides: `include` tags.

```

I think a paragraph break helps readability here.

> To demonstrate this, we will save the HTML snippet for creating navigation links into

Reviewer: ```suggestion
To demonstrate this, we will save an HTML snippet for creating navigation links into
```

a new file called `navigation.html` in a new folder called `_includes` within our repository. The folder `_includes` has special
meaning to Jekyll - it tells Jekyll where to look for code snippets that can be reused (included) in other parts of the website. 

1. Click "Create new file" under the "Add file"
dropdown on your repository homepage
2. In the "Name your file..." box, type `_includes/`. As you enter the "/" after the folder name "_includes", 
the folder is automatically inserted in the path displayed in front of the box for naming the file you are adding.
3. You can then name the file `navigation.html` and, when you commit the changes,
   the `_includes` folder will have been added to your repository too.
4. Insert the following HTML snippet into `navigation.html` and commit the changes:

    ~~~
    {% raw %}<table>
    <tr>
    <td><a href=".">Home</a></td>
    <td><a href="about">About</a></td>
    </tr>
    </table>
    <hr>{% endraw %}
    ~~~
The snippet will create a table with a single row of links followed by a horizontal line separator. 

> 5. Now insert the following `_include` directive at the beginning of `index.md`:

Reviewer: ```suggestion
5. Now insert the following `include` directive at the beginning of `index.md`:
```

    ~~~
    {% raw %}---
    lesson-example: "https://carpentries.github.io/lesson-example/"
    ---
             
    {% include navigation.html %}
             

>     # {{ site.title }}

Reviewer: ```suggestion
    # Building Websites in GitHub
```

`{{ site.title }}` was removed in #241

> and using it in a page (`{% raw %}{{site.social}}{% endraw %}` for the example above).
{: .callout }

> ## Reusing Footer 

Reviewer: ```suggestion
## Reusable Footer 
```

> ## Exercise: Including Contact Information in Footer
>
> The last line of `index.md` includes the kind of information you might want to

> > re-use in multiple places throughout your site as a footer.

Reviewer: ```suggestion
> reuse in multiple places throughout your site as a footer.
```

to keep consistent with the changes made elsewhere in this PR.

> ~~~
> {: .language-markdown }
>

> > Convert the above Markdown snippet into HTML and save it into `_includes/footer.html` file,

Reviewer: ```suggestion
> Convert the above Markdown snippet into HTML and save it into a `_includes/footer.html` file,
```

> > then use the `include` directive to re-insert it

Reviewer: ```
> then use the `include` directive to re-insert it
```

If we are using "reuse" now instead of "re-use," do you think this should be "reinsert?"

Author: Sounds good - I have probably introduced a few more inconsistencies along the way - e.g. when we refer to "our repository" vs "your repository" interchangeably. The whole lesson would benefit from a language polish - will add a separate issue.

> at the bottom of your site's `index.md` page (making sure you remove the equivalent code). Also insert 
> it at the bottom of the `about.md` page in a similar manner. We will explain 

> > why we meed the file to be in HTML rather than Markdown shortly.

Reviewer: ```suggestion
> why we need the file to be in HTML rather than Markdown shortly.
```
